example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -199,6 +199,7 @@
 				97C146EC1CF9000F007C117D /* Resources */,
 				9705A1C41CF9048500538489 /* Embed Frameworks */,
 				3B06AD1E1E4923F5004D2608 /* Thin Binary */,
+				8C7C6621B70C9CF94E219643 /* [CP] Embed Pods Frameworks */,
```

Comment:
this should not be included since we are moving away from Cocoapods to use SPM, can you turn on SPM support and build the examples app again?

Reply:
Good catch -- updated in latest commit :)